tests/datasource/data_connector/test_data_connector_query_sql.py

```diff
@@ -258,7 +258,7 @@ def my_custom_batch_selector(batch_identifiers: dict) -> bool:
         datasource_name="taxi_multi_batch_sql_datasource",
         data_connector_name="by_pickup_date_time",
         data_asset_name="yellow_tripdata_sample_2020_01",
-        batch_identifiers=IDDict({"pickup_datetime": "2020-01-01 08"}),
+        batch_identifiers=IDDict({"pickup_datetime": "2020-01-01 23"}),
```

Comment:
These were incorrect previously even though these tests state that they are testing for sorting behavior!

Reply:
Thank you for finding this :)